supported-devices/index.js

```diff
@@ -35,32 +35,35 @@
 
 */
 
-
```

Comment:
Why are we changing 4 space tab substitution down to 2. Can we please change it back.

Reply:
Is there a reason for this?

Reply:
I think the linter was complaining about it

Reply:
Interesting how the linter would be complaining about this. I looked at the ecma standard and it said that the spacing needs to be consistent, and didn't specify 2 or 4 spaces. Doesn't make much sense to change it from the style more commonly used in the documentation

Reply:
IDE Formatting sometimes does this